tests/test_utils.py

```diff
@@ -624,6 +624,28 @@ def test_expand_file_path(tmpdir, index, root, expected):
     pd.testing.assert_index_equal(expanded_index, expected)
 
 
+@pytest.mark.parametrize(
+    'index, expected',
+    [
+        (
+            audformat.segmented_index(['f1'] * 2, [0, 1], [2, 3]),
```

Comment:
Could you please add tests for an empty index and a segmented index with two files where the segments across files overlap but not within files (to make sure we only detect overlap within a file).

Reply:
And maybe also a test without overlap but where the segments touch each other (we expect `False' in that case).

Reply:
> Could you please add tests for an empty index and a segmented index with two files where the segments across files overlap but not within files (to make sure we only detect overlap within a file).

Added.

> And maybe also a test without overlap but where the segments touch each other (we expect False in that case).

There is already a test case for that

Reply:
We usually leave it to the reviewer to unresolve the issues.

Reply:
> There is already a test case for that

Sorry, overlooked it.

> Added.

Just to be on the safe side, please add also one for an empty `filewise` index.

Reply:
> We usually leave it to the reviewer to unresolve the issues.

Okay didn't know that

> Just to be on the safe side, please add also one for an empty filewise index.
I will

Reply:
>  We usually leave it to the reviewer to unresolve the issues.

Just do double check, you mean resolve right ?

Reply:
Oh of course I mean "resolve" :) This way you give the reviewer the chance to check your proposed changes and if necessary complain again :)

Reply:
We usually still leave a comment for the reviewer like "suggestion applied" or something, so the reviewer knows that the issue has been tackled.

Reply:
> Oh of course I meant "resolve" :) This way you give the reviewer the chance to check your proposed changes and if necessary complain again :)

Haha okay, I get it :)

Reply:
suggestion applied :)